content/en/blog/2024/otel-collector-anti-patterns/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: OpenTelemetry Collector Anti-Patterns
+linkTitle: OTel Collector Anti-Patterns
+date: 2024-02-23
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep),
+
+canonical_url: https://adri-v.medium.com/43dca4a857a0
+---
+
+![House on stilts against ocean and mountain backdrop](house-on-stilts.jpg)
+
+The [OpenTelemetry Collector](/docs/collector) is one of my favorite
+OpenTelemetry (OTel) components. It’s a flexible and powerful data pipeline
+which allows you to ingest OTel data from one or more sources, transform it
+(including batching, filtering, and masking), and export it to one or more
+observability backends for analysis. It’s vendor-neutral. It’s extensible,
+meaning that you can create your own custom components for it. What’s there not
+to like?
+
+And like many tools out there, it is also very easy to fall into some bad
+habits. Today, I will dig into five OpenTelemetry Collector anti-patterns, and
+how to avoid them. Let’s get started!
+
+## Anti-Patterns
+
+### 1- Not using the OpenTelemetry Collector
```

Comment:
I wonder if this could come last. It has a nice bottom line message that reminds folks about the benefits of the Collector. Opening with it is a bit shocking, but also valid.

Reply:
@theletterf Moved to the end. Let me know what you think of the flow with its new spot.